src/Conda.jl

```diff
@@ -224,6 +224,21 @@ function list(env::Environment=ROOTENV)
     runconda(`list`, env)
 end
 
+"""
+List all packages and write them to an export file for use the Conda.create
```

Comment:
The standard docstring format begins with the function signature.  See https://docs.julialang.org/en/v1/manual/documentation/

Reply:
Sure. I was just trying to follow the convention already used in this file. https://github.com/JuliaPy/Conda.jl/blob/master/src/Conda.jl#L92